index.html

```diff
@@ -91,8 +92,8 @@ <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
                 Submit
               </button>
             </div>
-
           </form>
+        </section>
     </div>
 
     <script  src="js/index.js"></script>
```

Comment:
I suggest you remove blank lines in your code, adding unnecessary blank lines makes the code not look clean and it will cause linter errors. avoiding blank lines improves the readability of your code by others.